tools/dev/update_user_agent_strings.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+import requests
+import re
+
+def replace_agent_string(lines, replace_marker, url, regex):
+  VALID_CHARS = 'a-zA-Z0-9\\(\\);:\\.,/_ '
+  regex = regex.replace('{VALID_CHARS}', VALID_CHARS)
+  print(f'Updating {replace_marker}')
+  for x in range(0, len(lines)):
+    if replace_marker in lines[x]:
+      break
```

Comment:
```suggestion
  for line in lines:
    if replace_marker in line:
      break
```

Reply:
I think this will break the subsequent code when we re-insert it back into the list? We need the index reference to do in-place replacement (unless we also change it to rebuild the list from scratch too).